### PR TITLE
Verify cluster can scale down

### PR DESCRIPTION
**Pull request by deepthidevaki**
## Description

- Extracted common asserts to a `ClusterActuatorAssert`
- Extracted common utils for scale up and scale  down tests
- Added test for scale down

## Related issues

related #14953

### CODE REVIEW

File: qa/util/src/main/java/io/camunda/zeebe/qa/util/topology/ClusterActuatorAssert.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+import io.camunda.zeebe.management.cluster.PostOperationResponse;
+import io.camunda.zeebe.management.cluster.TopologyChange.StatusEnum;
+import io.camunda.zeebe.qa.util.actuator.ClusterActuator;
+import io.camunda.zeebe.qa.util.cluster.TestCluster;
+import java.time.OffsetDateTime;
+import org.assertj.core.api.AbstractObjectAssert;
+import org.assertj.core.api.Assertions;
+
+public final class ClusterActuatorAssert
```
Comment (oleschoenburg) [inline qa/util/src/main/java/io/camunda/zeebe/qa/util/topology/ClusterActuatorAssert.java:18]: :+1:

**Review by lenaschoenburg — APPROVED** [review]
Nice cleanup of the test utilities :+1:

**Comment by deepthidevaki** [comment]
bors merge

**Comment by deepthidevaki** [comment]
bors merge

**Pull request merged**